environ/test/environ/core_test.clj

```diff
@@ -25,4 +25,8 @@
   (testing "env file with irregular keys"
     (spit ".lein-env" (prn-str {:foo.bar "baz"}))
     (let [env (refresh-env)]
-      (is (= (:foo-bar env) "baz")))))
+      (is (= (:foo-bar env) "baz"))))
+  (testing "env file with code"
+    (spit ".lein-env" "{:foo #=(str \"bar \" \"baz \")}")
+    (is (thrown-with-msg? IllegalArgumentException #".lein-env is not valid EDN data" (refresh-env)))))
```

Comment:
Somehow this isn't catching the exception that's being thrown, maybe because of the eval craziness at the top of the file? Any ideas there?

Everything's right about the error message, I just don't seem to be able to test for it.